api/v1/qdrantcluster_types.go

```diff
@@ -913,6 +950,53 @@ type NodeStatus struct {
 	// Version specifies the version of Qdrant running on the node
 	// +optional
 	Version string `json:"version,omitempty"`
+
+	// Status phase of the Pod of the node
+	// +optional
+	PodPhase corev1.PodPhase `json:"podPhase,omitempty"`
+	// Conditions of the Pod of the node
+	// +optional
+	PodConditions []corev1.PodCondition `json:"podConditions,omitempty"`
+	// Status message of the Pod of the node
+	// +optional
+	PodMessage string `json:"podMessage,omitempty"`
+	// Status reason of the Pod of the node
+	// +optional
+	PodReason string `json:"podReason,omitempty"`
+	// Details container statuses of the Pod of the node
+	// +optional
+	ContainerStatuses []corev1.ContainerStatus `json:"containerStatuses,omitempty"`
+	// Recent Kubernetes Events related to the Pod of the node
+	// Events that happened in the last 30 minutes are stored.
+	// +optional
+	PodEvents []KubernetesEventInfo `json:"events,omitempty"`
+
+	// The number of times the main qdrant container has been restarted.
+	// +optional
+	RestartCount int32 `json:"restartCount,omitempty"`
+
+	// Status of the database storage PVC
+	// +optional
+	DatabasePVCStatus NodePVCStatus `json:"databasePVCStatus,omitempty"`
+	// Status of the snapshots storage PVC
+	// +optional
+	SnapshotsPVCStatus NodePVCStatus `json:"snapshotsPVCStatus,omitempty"`
+}
+
+type NodePVCStatus struct {
```

Comment:
Nice. We can use this to better report the status after changing VolumeAttributeClassName.